src/app/(main)/(user)/borrow-form/page.tsx

```diff
@@ -68,19 +98,19 @@ const LMBInventoryForm = () => {
   };
 
   return (
-    <div className="flex w-screen h-screen justify-center items-center bg-gray-100">
-      <Card className="p-8 w-full max-w-[935px] max-h-[700px] shadow-lg">
+    <div className="flex justify-center items-center bg-gray-100 w-full h-screen">
+      <Card className="my-3 pt-8 px-8 pb-4 lg:w-3/5 md:w-4/5 w-full h-[610px] shadow-lg md:rounded-lg rounded-none">
```

Comment:
check latest in main from biological inventory form for dimensions, copy paste that